cypress/support/step_definitions/common/common.ts

```diff
@@ -73,3 +73,49 @@ After({ tags: "@organisation" }, () => {
     }
   });
 });
+
+Before({ tags: "@ciel" }, () => {
+  cy.createOrganisation("CIEL", true)
+   .createOrgSource("CIEL", "CIEL", true)
+   .createConcept(
+      [
+       {
+          name: "Serum",
+          locale: "en",
+          locale_preferred: true,
+          name_type: "FULLY_SPECIFIED"
+        }
+      ],
+      "/orgs/CIEL/sources/CIEL/",
+      "1001"
+    )
+    .createConcept(
+      [
+        {
+          name: "Whole blood sample",
+          locale: "en",
+          locale_preferred: true,
+          name_type: "FULLY_SPECIFIED"
+        }
+      ],
+      "/orgs/CIEL/sources/CIEL/",
+      "1000"
+    )
+    .createConcept(
+      [
+        {
+          name: "Plasma",
+          locale: "en",
+          locale_preferred: true,
+          name_type: "FULLY_SPECIFIED"
+        }
+      ],
+      "/orgs/CIEL/sources/CIEL/",
+      "1002"
+    );
+});
+
+After({ tags: "@ciel" }, () => {
+ cy.deleteOrgSource("CIEL", "CIEL", true).deleteOrganisation("CIEL", true);
+});
+
```

Comment:
For now, let's not change common.ts. We can do this as a new ticket at some point.

Reply:
@ibacher I have removed the proposed content